cdap-ui/app/directives/widget-container/widget-schema-editor/widget-schema-editor.js

```diff
@@ -5,10 +5,13 @@ angular.module(PKG.name + '.commons')
       scope: {
         model: '=ngModel',
         config: '=',
+        plugin: '=',
```

Comment:
Right now this is a little mis-leading. plugin here is actually plugin properties and config is schema properties. Is it possible to use the same name or something that signifies what is being passed?